### PR TITLE
Add signing for .js files

### LINKED ISSUE #12744: Sign the .js files
main.js, settings.js

### PR DESCRIPTION
**Pull request by sean-mcmanus**
Fixes https://github.com/microsoft/vscode-cpptools/issues/12744

This is based on https://github.com/microsoft/vscode-cmake-tools/pull/3573

### CODE REVIEW

File: Build/signing/SignFiles.proj [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  <PropertyGroup>
+    <BaseOutputDirectory>$(BUILD_STAGINGDIRECTORY)/Extension</BaseOutputDirectory>
+    <!-- These properties are required by MicroBuild, which only signs files that are under these paths -->
+    <IntermediateOutputPath>$(BaseOutputDirectory)</IntermediateOutputPath>
+    <OutDir>$(BaseOutputDirectory)</OutDir>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Because of Webpack bundling, these are the only shipping Javascript file.
```
Comment (bobbrow) [inline Build/signing/SignFiles.proj]: [sp] file -> files

**Comment by bobbrow** [comment]
The PR you linked to also has a proj for signing the vsix. Do you plan to add that one as well?

**Comment by sean-mcmanus** [comment]
> The PR you linked to also has a proj for signing the vsix. Do you plan to add that one as well?

Yeah, I'm saving that for a follow up PR tracked by https://github.com/microsoft/vscode-cpptools/issues/12725.

**Review by bobbrow — APPROVED** [review]

File: Build/signing/SignFiles.proj (hunk 1 above)
Comment (sean-mcmanus) [reply Build/signing/SignFiles.proj]: fixed

**Pull request merged**